docs/product/components/breadcrumbs.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Breadcrumbs
+description: Breadcrumbs are used to provide context for the currently-viewed page.
+---
+<section class="stacks-section">
+    {% header h2 | Example %}
+    <p class="stacks-copy">Breadcrumbs show users where they are within a site’s hierarchy. Breadcrumbs help orient the user and allow for navigation to previous page levels. It can be appropriate to use a breadcrumb when:</p>
+    <ul class="stacks-list">
+        <li>The user is many levels deep</li>
+        <li>The current page does not have its own navigation</li>
+        <li>The user needs to quickly go back to the previous parent page</li>
+    </ul>
+    <p class="stacks-copy">Breadcrumbs should be the first page element—placed directly above the page’s title. The page the user is on should not appear in the breadcrumb trail.</p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<nav class="s-breadcrumb mb8 sm:mb4">
```

Comment:
Shouldn't the component be called `s-breadcrumbs` instead of `s-breadcrumb`?

Reply:
In the systems that I surveyed, it goes both ways, but a singular breadcrumb is more common. I thought we should keep our internal logic of the component itself is just called a breadcrumb, while the pages referencing them are plural. This follows with "Avatars" being `s-avatar`, "Notices" being `s-notice`, and so on.

Reply:
But isn't a "singular breadcrumb" that thing that you call `s-breadcrumb--item`?

Reply:
Yep, indeed, no arguing that! But I think the internal logic ought to apply here. Design systems are split on this, and I've chosen `s-breadcrumb` because all our components are each singular in name. "You'll need to add a breadcrumb there".

Happy to change to `s-breadcrumbs` if it's bugging everyone too much.

Reply:
Counter-example: `s-tabs` 😄 

Yeah, my vote is for `s-breadcrumbs`